hack/jenkins/e2e.sh

```diff
@@ -806,6 +806,59 @@ case ${JOB_NAME} in
           )"}
     ;;
 
+  kubernetes-e2e-gke-trusty-prod)
+    # Override GKE default to use prod Cloud SDK and prod endpoint
+    CLOUDSDK_BUCKET=""
+    GKE_API_ENDPOINT="https://container.googleapis.com/"
+    : ${E2E_CLUSTER_NAME:="jkns-gke-e2e-prod-trusty"}
+    : ${E2E_NETWORK:="e2e-gke-trusty-prod"}
+    : ${E2E_SET_CLUSTER_API_VERSION:=y}
+    : ${JENKINS_USE_SERVER_VERSION:=y}
+    : ${PROJECT:="kubekins-e2e-gke-trusty-prod"}
+    : ${FAIL_ON_GCP_RESOURCE_LEAK:="true"}
+    : ${GINKGO_TEST_ARGS:="--ginkgo.skip=$(join_regex_allow_empty \
+          ${GKE_DEFAULT_SKIP_TESTS[@]:+${GKE_DEFAULT_SKIP_TESTS[@]}} \
+          ${GCE_DEFAULT_SKIP_TESTS[@]:+${GCE_DEFAULT_SKIP_TESTS[@]}} \
+          ${GCE_FLAKY_TESTS[@]:+${GCE_FLAKY_TESTS[@]}} \
+          ${TRUSTY_STABLE_SKIP_TESTS[@]:+${TRUSTY_STABLE_SKIP_TESTS}} \
+          )"}
+    ;;
+
+  kubernetes-e2e-gke-trusty-staging)
+    # Override GKE default to use rc Cloud SDK and staging endpoint
+    CLOUDSDK_BUCKET="gs://cloud-sdk-build/testing/rc"
+    GKE_API_ENDPOINT="https://staging-container.sandbox.googleapis.com/"
+    : ${E2E_CLUSTER_NAME:="jkns-gke-e2e-staging-trusty"}
+    : ${E2E_NETWORK:="e2e-gke-trusty-staging"}
+    : ${E2E_SET_CLUSTER_API_VERSION:=y}
+    : ${JENKINS_USE_SERVER_VERSION:=y}
+    : ${PROJECT:="e2e-gke-trusty-staging"}
```

Comment:
You mean the `PROJECT` name? If so, it's intended. GCE has a limit of 30 chars on project names, and `kubekins-gke-e2e-trusty-staging` is over the limit so I decided to trim it. You are right it's not analogical to others, but I don't think it matters, and this is the project that I have credentials, quotas and other acls set up.